webui/react/src/components/CheckpointDetails.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { PropsWithChildren, useCallback } from 'react';
+
+import useModalCheckpoint from 'hooks/useModal/useModalCheckpoint';
+import {
+  CheckpointDetail, CheckpointWorkloadExtended, ExperimentBase,
+} from 'types';
+
+interface Props {
+  checkpoint: CheckpointWorkloadExtended | CheckpointDetail;
+  experiment: ExperimentBase;
+  title: string;
+}
+
+const CheckpointDetails: React.FC<Props> = (
+  {
+    checkpoint,
+    experiment,
+    title,
+    ...props
```

Comment:
**nit:** could probably just specify `children` since it's the only `props` being used